code/__DEFINES/combat.dm

```diff
@@ -39,8 +39,7 @@
 #define IGNORESLOWDOWN	256
 #define GODMODE		4096
 #define FAKEDEATH	8192	//Replaces stuff like changeling.changeling_fakedeath
-#define DISFIGURED	16384	//I'll probably move this elsewhere if I ever get wround to writing a bitflag mob-damage system
```

Comment:
every time we kill off a bitflag, I become *slightly* happier.

Reply:
I agree so much.